### PR TITLE
Adds utils export

### PR DESCRIPTION
**Pull request by douglance**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export { CALLDATA as Calldata } from './calldata/index';
 export { Client } from './client';
 export { EXTERNAL as Constants } from './constants';
+export * from './util'
```
Comment (alex-miller-0) [inline src/index.ts:4]: I'd like to mimic what we are doing with `Constants` in the line above. I find it cleaner to have all utils organized in one export. Users can always reconstruct.

**Review by alex-miller-0 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (douglance) [reply src/index.ts]: [bd00fbc](https://github.com/GridPlus/gridplus-sdk/pull/371/commits/bd00fbcd9bf6a01bd793069953741b78247ba32e)

**Comment by douglance** [comment]
@alex-miller-0 [2ebeb29](https://github.com/GridPlus/gridplus-sdk/pull/371/commits/2ebeb2963fb1f163d01df0352957e736a3bab6b3) util.ts now uses both bn and bignumber to ensure compat.

File: src/util.ts [hunk 2]
```diff
@@ -290,3 +294,81 @@ export const randomBytes = function(n) {
...
 }
+
+/**
+ * Generates a private key for use in maintaining connection to device.
+ * @param {Buffer} deviceId - The device ID of the device you want to generate a token for.
+ * @param {Buffer} password - The password entered when connecting to the device.
+ * @param {Buffer} appName - The name of the application.
+ * @returns a private key as a Buffer
+ */
+export const generatePrivateKey = (
```
Comment (alex-miller-0) [inline src/util.ts:305]: This name is too generic. Please change to `generateAppSecret` or something.

Comment (douglance) [reply src/util.ts]: [123bf7d](https://github.com/GridPlus/gridplus-sdk/pull/371/commits/123bf7d5f7a9e03b3851717dcbe3ec1b1fb70c29)

**Review by alex-miller-0 — APPROVED** [review]

**Pull request merged**